packages/dashmate/src/templates/renderServiceTemplatesFactory.js

```diff
@@ -18,25 +17,20 @@ function renderServiceTemplatesFactory() {
   function renderServiceTemplates(config) {
     dots.templateSettings.strip = false;
 
-    const templatesPath = path.join(__dirname, '..', '..', 'templates');
-
     const templatePaths = glob
-      .sync(`${templatesPath}/**/*.dot`)
+      .sync(`${TEMPLATES_DIR}/**/*.dot`)
       // Do not render platform templates if it's not configured
       .filter((templatePath) => (
         !templatePath.includes('templates/platform') || config.get('platform.enable')
       ));
```

Comment:
Do we still need this?

Reply:
Yes, why not?

Reply:
I mean currently all configs have `platform` section. If we remove delete then when we enable it we create files and then we disable it and we keep old version of templates. I think if we do not remove previous templates we should render all of them every time. What are your thoughts?